Add script to compute manifest of dcrinstall manifests

jrick opened the pull request:
URLs of the manifests dcrinstall may download and verify are read from
a separate file maintained in this repo.

marcopeereboom reviewed (changes requested):
I think the `mv` can go since we are going to symlink. Having the `.asc` extension is more appropriate.

jrick committed:
Add script to compute manifest of dcrinstall manifests

URLs of the manifests dcrinstall may download and verify are read from
a separate file maintained in this repo.

The manifest created by this script is signed with `gpg --clearsign`
to produce a single file that can be linked to for dcrinstall to
always fetch latest.
